deny.toml

```diff
@@ -80,8 +83,6 @@ skip = [
   { name = "windows_x86_64_gnullvm", version = "0.42.2" },
   # windows-targets
   { name = "windows_x86_64_msvc", version = "0.42.2" },
-  # cpp_macros
-  { name = "aho-corasick", version = "0.7.20" },
```

Comment:
Why do you remove `aho-corasick`? It's makes the CI fail because we use two versions of it.

Reply:
Strange, I got a warning that the rule wasn't used. I put it back.

Reply:
You have to run `cargo deny --all-features check`.

Reply:
Oh I forgot about that